frontend/src/Products/index.jsx

```diff
@@ -94,8 +83,8 @@ class ProductsWidget extends Component {
    */
   shouldComponentUpdate(nextProps, nextState) {
```

Comment:
I wonder how you could commit this. `nextState` is not used. Please remove this parameter.